.github/workflows/run_rc_validation.yml

```diff
@@ -81,8 +81,14 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ env.RC_TAG }}
-      - name: Setup GitHub CLI
-        uses: ./.github/actions/setup-gh-cli-linux
+      - id: setup-gh-cli-linux
```

Comment:
could you revert it to use the `./.github/actions/setup-gh-cli-linux` action please?

Reply:
Sure, the change was for testing purposes but now I will revert it.